app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'AuthorId'
+  belongs_to :post, class_name: 'Post', foreign_key: 'PostId'
+
+  after_save :update_counter
+
+  def update_counter
+    post.CommentsCounter.nil? ? post.CommentsCounter = 1 : post.CommentsCounter += 1
+    post.update(CommentsCounter: post.CommentsCounter)
```

Comment:
- [ ] [OPTIONAL] I like the way you came up with these lines of code here, but have you thought about this Ruby helper method called `increment!`? Something like this: `author.increment!(:comments_counter)`